### PR TITLE
Fixes/rewrites on videos.php

### PR DESCRIPTION
**Pull request by Quix0r**
Fixes/rewrites:
- use empty()/isset() instead of deprecated x()
- merged 2 nested if() blocks into one
- avoided nested if() block inside else block by rewriting it to elseif()
- $contact_id is an integer, let's test on > 0 here
- added a lot spaces and some empty lines for better readability

### CODE REVIEW

File: mod/videos.php [hunk 1]
```diff
@@ -192,15 +194,15 @@ function videos_content(App $a) {
 	// videos/name/video/xxxxx/edit
 
 
-	if((Config::get('system','block_public')) && (! local_user()) && (! remote_user())) {
+	if ((Config::get('system','block_public')) && (! local_user()) && (! remote_user())) {
```
Comment (MrPetovan) [inline mod/videos.php]: Please remove space after `!` operator.

File: mod/videos.php [hunk 2]
```diff
@@ -241,68 +242,67 @@ function videos_content(App $a) {
...
+				$visitor = $contact_id;
 			}
 		}
 	}
 
 	$groups = [];
 
 	// perhaps they're visiting - but not a community page, so they wouldn't have write access
-	if(remote_user() && (! $visitor)) {
+	if (remote_user() && (! $visitor)) {
```
Comment (MrPetovan) [inline mod/videos.php]: Please remove space after `!` operator.

File: mod/videos.php [hunk 3]
```diff
@@ -241,68 +242,67 @@ function videos_content(App $a) {
...
-			$contact_id = $_SESSION['cid'];
-			$contact = $a->contact;
-		}
+	if (!$remote_contact && local_user()) {
+		$contact_id = $_SESSION['cid'];
+		$contact = $a->contact;
 	}
 
-	if($a->data['user']['hidewall'] && (local_user() != $owner_uid) && (! $remote_contact)) {
+	if ($a->data['user']['hidewall'] && (local_user() != $owner_uid) && (! $remote_contact)) {
```
Comment (MrPetovan) [inline mod/videos.php]: Please remove space after `!` operator.

**Comment by Quix0r** [comment]
OT: Eh. is `EOL` okay? Shouldn't it be `PHP_EOL`?

**Comment by MrPetovan** [comment]
Is it defined anywhere?

**Comment by Quix0r** [comment]
Yes, `boot.php`. Okay, just a bit confusing. And I think I accidentally replaced one somewhere else. :-/

**Comment by MrPetovan** [comment]
Did you run theses changes on your system?

**Comment by Quix0r** [comment]
Yes, also this is now online. My `develop` branch always contains all changes from all my PR branches.

**Pull request merged**